src/suggestions.js

```diff
@@ -57,8 +58,8 @@ export function getSuggestedQuery(element, variant) {
     return makeSuggestion('PlaceholderText', placeholderText, {variant})
   }
 
-  const textContent = normalize(element.textContent)
-  if (textContent) {
+  const textContent = normalize(getNodeText(element))
+  if (textContent && !element.matches('script, style')) {
```

Comment:
Can't we use the same ignores that are used by the query options? I think that would be best

Reply:
done.